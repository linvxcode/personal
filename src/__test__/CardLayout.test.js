import 'intersection-observer'

import React from "react";
import { render, screen } from "@testing-library/react";
import CardLayout from "../Component/CardLayout";

describe('Card Layout', () => {
    it('Render Card Layout' , () => {
        const {Card} = render(<CardLayout />);
        const text = screen.getByTestId('CardLayout');
        expect(text).toBeInTheDocument();
        expect(Card).toMatchSnapshot();

    })
})

