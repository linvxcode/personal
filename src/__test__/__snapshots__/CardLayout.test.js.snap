// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Layout Render Card Layout 1`] = `undefined`;
